### PR TITLE
Add Product Manager job ad

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jamesshedden : can you help me set a new section for Product? I added it in routes.js as Engineering, but it would be nice to have a separate Product section.

FYI: @jesssummerfield @duncanbarrigan

### CODE REVIEW

File: app/pages/about/jobs/positions/product-manager.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            dictating detailed answers.
+          </li>
+          <li>
+            How we build it: We believe in a collaborative partnership where the Product team focuses on outcomes, while Engineering
+            focuses on how we get there. Product’s main role in execution is to be the voice of the market and to make sure we achieve
+            what we set out to do.
+          </li>
+          <li>
+            Communicating with everyone: Product is a key part of the glue that binds together teams at GoCardless. We communicate the
+            who, what, why and when” to the company. We also work closely with Marketing, Sales and Support on how best to communicate
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/product-manager.js]: @gabrielamatias should be when not when"

File: app/pages/about/jobs/positions/product-manager.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            what’s happening to our customers.
+          </li>
+        </ul>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          As a Product Manager you will work closely with one of our  Engineering Managers to lead one of the development teams at GoCardless.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/product-manager.js]: @gabrielamatias think there might be a double space between 'our' and 'Engineering'

File: app/pages/about/jobs/positions/product-manager.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            Product. When you’re presented with data, analysis should be a reflex.
+          </li>
+        </ul>
+        <p className='para'>
+          Candidates should have a proven record of success in a challenging environment,
+          which could be anything from software engineering through entrepreneurship to strategy consulting.
+        </p>
+        <p className='para'>
+            In your application, please include your CV and a cover letter
+            explaining why you are a great candidate for this position.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/product-manager.js]: @gabrielamatias can we add the bit on diversity? Think it was in the initial google doc but might have got lost!

**Comment by jesssummerfield** [comment]
Couple of small changes but all good apart from that!

**Comment by jamesshedden** [comment]
@gabrielamatias Spoke to you about this yesterday already but just to confirm I pushed a commit to add the `Product` section 😄 

Let me know if you need help with anything else! ⚡️

**Pull request merged**